targets/targets.json

```diff
@@ -2740,6 +2740,58 @@
         ],
         "device_name": "STM32G071RBTx",
         "bootloader_supported": true
+    },
+	"NUCLEO_G031K8": {
+        "inherits": [
+            "MCU_STM32_BAREMETAL"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
```

Comment:
Remove ARDUINO
It means here Arduino Uno

Reply:
Removed both in e74a2bc3714cee7c4960c6065d6e6f2aee01e162 as the NUCLEO_G031K8 does not support either morpho or Arduino Uno form factors.